packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -364,28 +375,76 @@ const {
   diff: { enabled }
 } = useInjectedViewerInterfaceState()
 
+const map: Record<ViewerKeyboardActions, [ModifierKeys[], string]> = {
+  [ViewerKeyboardActions.ToggleModels]: [[ModifierKeys.Shift], 'm'],
+  [ViewerKeyboardActions.ToggleExplorer]: [[ModifierKeys.Shift], 'e'],
+  [ViewerKeyboardActions.ToggleDiscussions]: [[ModifierKeys.Shift], 't'],
+  [ViewerKeyboardActions.ToggleFilters]: [[ModifierKeys.Shift], 'f'],
```

Comment:
Shift+f does nothing and I don't even see any kind of component in the template for this, so I've no idea what this is even supposed to be

Reply:
Sorry, you're right. I copied the list from the `type ActiveControl` which includes a filters. But I've removed the shortcuts to this now.